content/cloud-docs/api-docs/configuration-versions.mdx

```diff
@@ -44,6 +44,22 @@ You need read runs permission to list and view configuration versions for a work
 
 [permissions-citation]: #intentionally-unused---keep-for-maintainers
 
+## Attributes
+
+### Configuration Version States
+
+The configuration version state is found in `data.attributes.status`, and you can reference the following list of possible states.
+
+A configuration version created through the API can only be used in runs if it is in an `uploaded` state. A configuration version created through a linked VCS repository may also be used in runs if it is in an `archived` state.
```

Comment:
should this be "through the API or CLI"? Just want to make sure we're covering all workflows!

Reply:
Yes it should! Thank you 🙌🏻